tests/management/group/test_view.py

```diff
@@ -1768,6 +1795,71 @@ def test_system_flag_update_on_add(self, send_kafka_message):
             self.assertTrue(self.defGroup.system)
             self.assertEqual(self.defGroup.roles().count(), 1)
             response = client.post(url, test_data, format="json", **self.headers)
+            actual_call_arg = mock_method.call_args_list[0][0][0]
+            to_remove = actual_call_arg["relations_to_remove"]
+            to_add = actual_call_arg["relations_to_add"]
+
+            binding_mapping = BindingMapping.objects.get(
+                role=default_role, resource_type_name="workspace", resource_id=self.default_workspace.id
+            )
+
+            # New platform default role for tenant created
+            custom_default_group = Group.objects.get(platform_default=True, tenant=self.tenant)
+
+            def assert_group_tuples(tuple_to_replicate):
```

Comment:
Is this reused? Why define a method and then call it? Can we do the asserts directly?

Reply:
yes, probably there was some evolution - now it is not needed, removed 👍